internal/service/networkcontainer/resource_network_container_migration_test.go

```diff
@@ -0,0 +1,82 @@
+package networkcontainer_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
+)
+
+func TestAccMigrationNetworkContainerRS_basicAWS(t *testing.T) {
+	var (
+		projectName = acctest.RandomWithPrefix("test-acc")
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acc.PreCheckBasic(t) },
+		CheckDestroy: checkDestroy,
+		Steps: []resource.TestStep{
+			{
+				ExternalProviders: mig.ExternalProviders(),
+				Config:            configAWS(projectName, orgID, cidrBlock, providerNameAws, "US_EAST_1"),
+				Check: resource.ComposeTestCheckFunc(
+					checkExists(resourceName, &container),
+					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
+					resource.TestCheckResourceAttr(resourceName, "provider_name", providerNameAws),
+					resource.TestCheckResourceAttrSet(resourceName, "provisioned"),
+				),
+			},
+			mig.TestStep(configAWS(projectName, orgID, cidrBlock, providerNameAws, "US_EAST_1")),
```

Comment:
I reccomend to extract to var section:   
config = configAWS(projectName, orgID, cidrBlock, providerNameAws, "US_EAST_1")

Reply:
Done!